src/components/20-molecules/top-content-bar/index.scss

```diff
@@ -0,0 +1,39 @@
+.m-top-content-bar {
+  display: block;
+
+  &__container {
+    @include typo(primary, small, medium);
+
+    background-color: $color-sec-ocean-blue;
+    color: $color-prim-white;
+
+    min-height: 50px;
+    vertical-align: middle;
+
+    text-align: center;
+
+    padding-top: 5px;
+    padding-bottom: 5px;
+
+    box-sizing: border-box;
+
+    axa-container {
```

Comment:
A container inside a container? Looks like a code smell.

Reply:
its ok, is wanted/needed